elevate-survey/constants/routes.js

```diff
@@ -70,6 +80,236 @@ module.exports = {
 				type: 'GET'
 			},
 		},
+		{
```

Comment:
@praveenKDass make sure all api calls which accept request body their method is post

Reply:
handled api methods @VISHNUDAS-tunerlabs